airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -295,16 +310,34 @@ export const useFrequencyDropdownData = (
       }
     }
 
-    return frequencies.map((frequency) => ({
+    const basicFrequencies = frequencies.map((frequency) => ({
       value: frequency,
-      label: frequency
-        ? formatMessage(
-            {
-              id: `form.every.${frequency.timeUnit}`,
-            },
-            { value: frequency.units }
-          )
-        : formatMessage({ id: "frequency.manual" }),
+      label: formatMessage(
+        {
+          id: `form.every.${frequency.timeUnit}`,
+        },
+        { value: frequency.units }
+      ),
     }));
+
+    // Add Manual and Custom to the frequencies list
+    const customFrequency = formatMessage({
+      id: "frequency.cron",
+    });
+    const manualFrequency = formatMessage({
+      id: "frequency.manual",
+    });
+    const otherFrequencies = [
+      {
+        label: customFrequency,
+        value: customFrequency.toLowerCase(),
+      },
+      {
+        label: manualFrequency,
+        value: manualFrequency.toLowerCase(),
+      },
+    ];
```

Comment:
Can you swap the order of these so that `Manual` comes first before `Cron`? I think we always want to have manual at the top